cqlsh_tests/test_cqlsh.py

```diff
@@ -2429,6 +2430,33 @@ def test_cjk_output(self):
 """
         assert stdout_lines_sorted.find(expected) >= 0
 
+    @since('4.0')
+    def test_no_file_io(self):
```

Comment:
I know it's just a port but this could use a case where the same command but without the additional parameter doesn't error out.

Reply:
Yeah I think it's worth adding an additional patch that does this. I'll take a look.